### PR TITLE
 Adds sanity to most cyborg/mommi verbs

### PR DESCRIPTION
**Pull request by jknpj**
* Adds an incapacitated() check to name pick
* Adds an isDead() check to show alerts and also links it to the binary comms component
* Adds an isDead() check to show manifest and also links it to the binary comms component
* Adds an isDead() check to toggle holomap and also links it to the power cell component
* Adds an isDead() check to toggle component
* Adds an isDead() check to unlock cover
* Adds an incapacitated() check to sensor mode pick
* Adds an incapacitated() check to mommi hide
* Adds an incapacitated() check to mommi parking brake

Also moves cyborg verbs to their own file because robots.dm is super messy. Show/State Laws left untouched so you can still state'em in the middle of a flash and bash routine.

### CODE REVIEW

**Comment by Pathid** [comment]
Tying the holomap toggle to the camera component doesn't seem right. Wouldn't the holomap be internal? Secondly, what happens if the camera gets destroyed while the holomap is active?

**Comment by jknpj** [comment]
@Pathid 
Nothing. Even RIGHT NOW you get stuck like this most likely due holomap sanity preventing you from fucking with it while dead. If you just disable your camera you can still fuck with it. I can however, fix that.

ded

**Comment by Pathid** [comment]
Yes, but what is the logic of making it dependent on the camera?

(This may be neither here nor there but even handheld portable station maps don't have sanity regarding blindness.)

File: code/modules/mob/living/silicon/robot/robot_verbs.dm [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+	station_holomap.toggleHolomap(src)
+
+/mob/living/silicon/robot/verb/self_diagnosis_verb()
+	set category = "Robot Commands"
+	set name = "Self Diagnosis"
+
+	if(!can_diagnose())
+		to_chat(src, "<span class='warning'>Your self-diagnosis component isn't functioning.</span>")
+
```
Comment (DamianX) [inline code/modules/mob/living/silicon/robot/robot_verbs.dm]: I do believe a return is needed here

**Pull request merged**